tests/common/predictions.py

```
@@ -0,0 +1,311 @@
+import pandas as pd
+from sklearn.base import BaseEstimator
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.isotonic import IsotonicRegression
+from sklearn.model_selection import KFold
+
+CLIP = 1e-3
+
+
+def treatment_and_outcome_predictions(
+    data,
+    ps_model_dict: dict = {"model": BaseEstimator, "kwargs": {}, "calibrate": True},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)